src/imio/helpers/pdf.py

```
@@ -15,6 +18,29 @@
+    /Root, so any page access raises KeyError('/Root'). Recover by locating the
+    /Catalog object and pointing /Root at it.
+    """
+    reader = PdfReader(stream)
+    if "/Root" in reader.trailer:
+        return reader
+    for gen, idmap in reader.xref.items():
+        for idnum in idmap:
+            try:
+                obj = IndirectObject(idnum, gen, reader).get_object()
+            except Exception:
+                continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)